### ISSUE TITLE
Staging Review finding: Unclear to screen reader users when they've started or ended the flow

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#99004]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Accredited Representation Management
**Product:** Appoint a Representative MVP
**Feature:** Appoint a Representative + Representative Status Widget

## Findings details
**VA.gov Experience Standard - issue:** [User has enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** 

### Description

When moving from /appoint-rep/introduction to [/appoint-rep/claimant-type](https://staging.va.gov/get-help-from-accredited-representative/appoint-rep/claimant-type), nothing is announced to screen reader users to indicate that they've started their application.

Likewise, when moving from /appoint-rep/review-and-submit to [/appoint-rep/confirmation](https://staging.va.gov/get-help-from-accredited-representative/appoint-rep/confirmation), nothing is announced to screen reader users to indicate that they've successfully moved to the next page. 

And finally, when moving from /appoint-rep/confirmation to [/appoint-rep/next-steps](https://staging.va.gov/get-help-from-accredited-representative/appoint-rep/next-steps), nothing is announced.

Silence can be very disorienting, and leaves the user guessing: Did it work? Should I try to push the button again? Is the VA website broken? etc.

## Recommended action

This is one of those things that you get for free with traditional webpages, which trigger an announcement from screen readers every time the browser reloads. In an SPA where the browser never actually reloads, you have to handle this more proactively through focus management. When keyboard focus is set to a non-interactive element (using `tabindex="-1"), that prompts a screen reader to announce that content.

The forms library takes care of this out of the box by sending focus to the step indicator. But at points in your flow where you're doing something different than default forms library behavior, you need to make sure you're doing some kind of focus management.

Typically, you want to send focus to the first heading on the page that's descriptive of the content of the page and can be distinguished from the page you were previously on. I think that's going to be:
- On /appoint-rep/claimant-type, send focus to "Tell us who you are"
- On /appoint-rep/confirmation, send focus to "Download, print, and sign your form"
- On /appoint-rep/next-steps, send focus to "Your next steps"

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
